src/terminal/Terminal.h

```diff
@@ -473,6 +476,8 @@ class Terminal
         return selection_ && selection_->state() != Selection::State::Waiting && selection_->contains(_coord);
     }
 
+    bool isHighlighted(CellLocation const& _cell) const noexcept;
```

Comment:
I think we do not need to cref small trivial structs in Funktion Parameters. Not using cref here is also more Linux/AMD64 ABI friendly (all pareters passed directly via CPU registers), avoiding unnecessary derefs

Reply:
Done